buildconfig/stubs/pygame/math.pyi

```diff
@@ -0,0 +1,303 @@
+    def __init__(self, x: float, y: float) -> None: ...
+    def __len__(self) -> int: ...
+    @overload
+    def __setitem__(self, key: int, value: float) -> None: ...
+    @overload
+    def __setitem__(
+        self, key: slice, value: Union[Sequence[float], Vector2, Vector3]
+    ) -> None: ...
+    @overload
+    def __getitem__(self, i: int) -> float: ...
+    @overload
+    def __getitem__(self, s: slice) -> List[float]: ...
+    def __add__(self, other: _SupportsVector2) -> Vector2: ...
+    def __radd__(self, other: _SupportsVector2) -> Vector2: ...
+    def __sub__(self, other: _SupportsVector2) -> Vector2: ...
+    def __rsub__(self, other: _SupportsVector2) -> Vector2: ...
+    @overload
+    def __mul__(self, other: _SupportsVector2) -> float: ...
+    @overload
+    def __mul__(self, other: float) -> Vector2: ...
+    def __rmul__(self, other: float) -> Vector2: ...
+    def __truediv__(self, other: float) -> Vector2: ...
+    def __rtruediv__(self, other: float) -> Vector2: ...
+    def __floordiv__(self, other: float) -> Vector2: ...
+    def __neg__(self) -> Vector2: ...
+    def __pos__(self) -> Vector2: ...
+    def __bool__(self) -> bool: ...
+    def __iadd__(self, other: _SupportsVector2) -> Vector2: ...
+    def __isub__(self, other: _SupportsVector2) -> Vector2: ...
+    @overload
+    def __imul__(self, other: _SupportsVector2) -> float: ...
+    @overload
+    def __imul__(self, other: float) -> Vector2: ...
+    def dot(self, other: _SupportsVector2) -> float: ...
+    def cross(self, other: _SupportsVector2) -> Vector2: ...
+    def magnitude(self) -> float: ...
+    def magnitude_squared(self) -> float: ...
+    def length(self) -> float: ...
+    def length_squared(self) -> float: ...
+    def normalize(self) -> Vector2: ...
+    def normalize_ip(self) -> None: ...
+    def is_normalized(self) -> bool: ...
+    def scale_to_length(self, value: float) -> None: ...
+    def reflect(self, other: _SupportsVector2) -> Vector2: ...
+    def reflect_ip(self, other: _SupportsVector2) -> None: ...
+    def distance_to(self, other: _SupportsVector2) -> float: ...
+    def distance_squared_to(self, other: _SupportsVector2) -> float: ...
+    def lerp(self, other: Vector2, value: float) -> Vector2: ...
+    def slerp(self, other: Vector2, value: float) -> Vector2: ...
+    def elementwise(self) -> VectorElementwiseProxy[Vector2]: ...
+    def rotate(self, angle: float) -> Vector2: ...
+    def rotate_rad(self, angle: float) -> Vector2: ...
+    def rotate_ip(self, angle: float) -> None: ...
+    def rotate_rad_ip(self, angle: float) -> None: ...
+    def rotate_ip_rad(self, angle: float) -> None: ...
+    def angle_to(self, other: _SupportsVector2) -> float: ...
+    def as_polar(self) -> Tuple[float, float]: ...
+    def from_polar(
+        self, polar_value: Union[Sequence[float], Tuple[float, float]]
+    ) -> None: ...
+    def copy(self) -> Vector2: ...
+    def project(self, other: _SupportsVector2) -> Vector2: ...
```

Comment:
thanks for adding typehints for the  `project` method. I didn't add them back then.